src/Formatter/CallableFormatter.php

```diff
@@ -0,0 +1,29 @@
+<?php
+namespace League\BooBoo\Formatter;
+
+use Exception;
```

Comment:
I usually try to avoid namespace using classes which are part of the "root" namespace. Extra noise...